docs/4.1. Behaviour - Registration.md

```diff
@@ -81,13 +81,13 @@ The following error conditions describe likely scenarios encountered by Nodes wh
 
 ### Node Encounters HTTP 200 On First Registration
 
-If a Node is restarted, its first action upon discovering a Registration API is to register its 'Node' resource. On first registration with a Registration API this SHOULD result in a '201 Created' HTTP response code. If a Node receives a 200 code in this case, a previous record of the Node can be assumed to still exist within the network registry. In order to avoid the registry-held representation of the Node's resources from being out of sync with the Node's view, an HTTP DELETE SHOULD be performed in this situation to explicitly clear the registry of the Node and any sub-resources. A new Node registration after this point SHOULD result in the correct 201 response code.
+If a Node is restarted, its first action upon discovering a Registration API is to register its 'Node' resource. On first registration with a Registration API this SHOULD result in a '201 Created' HTTP response code. If a Node receives a 200 code in this case, a previous record of the Node can be assumed to still exist within the network registry. In order to avoid the registry-held representation of the Node's resources from being out of sync with the Node's view, an HTTP `DELETE` SHOULD be performed in this situation to explicitly clear the registry of the Node and any sub-resources. A new Node registration after this point SHOULD result in the correct 201 response code.
```

Comment:
```suggestion
If a Node is restarted, its first action upon discovering a Registration API is to register its Node resource. On first registration with a Registration API this SHOULD result in a `201 Created` HTTP response code. If a Node receives a 200 code in this case, a previous record of the Node can be assumed to still exist within the network registry. In order to avoid the registry-held representation of the Node's resources from being out of sync with the Node's view, an HTTP `DELETE` SHOULD be performed in this situation to explicitly clear the registry of the Node and any sub-resources. A new Node registration after this point SHOULD result in the correct 201 response code.
```

Reply:
```suggestion
If a Node is restarted, its first action upon discovering a Registration API is to register its Node resource (`/self`). On first registration with a Registration API this SHOULD result in a 201 (Created) HTTP response code. If a Node receives a 200 code in this case, a previous record of the Node can be assumed to still exist within the network registry. In order to avoid the registry-held representation of the Node's resources from being out of sync with the Node's view, an HTTP `DELETE` SHOULD be performed in this situation to explicitly clear the registry of the Node and any sub-resources. A new Node registration after this point SHOULD result in the correct 201 response code.
```